### PR TITLE
mohitspawar frisco tx 

### PR DESCRIPTION
**Pull request by mohitspawar**
Adding a few places to eat in Frisco TX.

### CODE REVIEW

**Comment by githubteacher** [comment]
I've never been to Frisco, Tx before, can you recommend some places to visit.

File: mohitspawar-frisco-tx.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Frisco TX
+
+## Place to eat
+- Nerdvana
```
Comment (githubteacher) [inline mohitspawar-frisco-tx.md]: This place sounds interesting, do you have any recommendations for what to get?

File: mohitspawar-frisco-tx.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Frisco TX
+
```
Comment (githubteacher) [inline mohitspawar-frisco-tx.md:2]: Never been to Frisco, I really like 🌮 s and 🍻 any specific places I should go?

File: mohitspawar-frisco-tx.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+## Place to eat
+- Nerdvana
+- Hutchin's BBQ
+- The Red Lantern
```
Comment (githubteacher) [inline mohitspawar-frisco-tx.md]: This sounds like a place that has great 🍔 s, if not, where should I go to get one?

**Review by githubteacher — COMMENTED** [review]
I have some questions, can you make changes to your content before we 🚢 ?

File: mohitspawar-frisco-tx.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# Frisco TX
```
Comment (sadashivkulthe) [inline mohitspawar-frisco-tx.md:1]: Where are you from Basically?

**Review by sadashivkulthe — COMMENTED** [review]
I've Q. Can u answer?

File: mohitspawar-frisco-tx.md (hunk 2 above)
Comment (mohitspawar) [reply mohitspawar-frisco-tx.md:2]: Daddy's Tacos is a good taco place

File: mohitspawar-frisco-tx.md (hunk 1 above)
Comment (mohitspawar) [reply mohitspawar-frisco-tx.md]: Shrimp Diablo 🌶

**Review by cjcollins — APPROVED** [review]

**Pull request merged**